build.psm1

```diff
@@ -1104,11 +1108,11 @@ function Start-PSPackage {
     Write-Verbose "Packaging RID: '$Runtime'; Packaging Configuration: '$Configuration'" -Verbose
 
     # Make sure the most recent build satisfies the package requirement
-    if (-not $Script:Options -or                                ## Start-PSBuild hasn't been executed yet
-        -not $Script:Options.CrossGen -or                       ## Last build didn't specify -CrossGen
-        $Script:Options.Runtime -ne $Runtime -or                ## Last build wasn't for the required RID
-        $Script:Options.Configuration -ne $Configuration -or    ## Last build was with configuration other than 'Release'
-        $Script:Options.Framework -ne "netcoreapp1.0") ## Last build wasn't for CoreCLR
+    if (-not $Script:Options -or                                                    ## Start-PSBuild hasn't been executed yet
+        ((-not $Script:Options.CrossGen) -AND -not ($Runtime -match "-x86")) -or    ## Last build didn't specify -CrossGen OR CrossGen check skipped since it is not supported for x86
```

Comment:
The command will complete successfully with a warning message saying that CrossGen is not supported on x86 because of blah.

Triggering the error is bad because user won't know if he/she can continue to use the generated assemblies from this failed build, and then have to build again without -CrossGen. It's a waste of time and frustrating because the script should already know that it's bound to fail in this case.